packages/fxa-auth-db-mysql/scripts/read-security-events.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * This script takes the uid of a user and will print a security event object of that user
+ * from various tables like securityEvents, sessionTokens and devices tables from fxa database
+ */
+
+// Usage: node readSecurityEvents.js uid
+
+const log = {
+  trace: console.log,
+  error: console.error,
+  stat: console.log,
+  info: console.log
+} //eslint-disable-line no-console
+const DB = require('../lib/db/mysql')(log, require('../db-server').errors)
+const config = require('../config')
+
+const securityEventsObject = {}
+
+// This input of user's data can be modified according to the usecase
+const user = {
```

Comment:
For the future what might be nice for the script is to be able to fetch by email OR uid. For the future, don't worry about it now.